docs/1.getting-started/6.data-fetching.md

```diff
@@ -150,6 +150,7 @@ Read more about `useAsyncData`.
 - `data`: the result of the asynchronous function that is passed in.
 - `pending`: a boolean indicating whether the data is still being fetched.
 - `refresh`/`execute`: a function that can be used to refresh the data returned by the `handler` function.
+- `clear`: a function that can be used to empty the state of `data` (and prevent any currently pending requests from updating it)
```

Comment:
`data` on its own might be confusing as the scope is broader than one property, WDYT? What about '... to reset the state of the data request (and cancel any currently pending requests)'?

Reply:
we could be even more explicit:

```suggestion
- `clear`: a function that can be used to set `data` to undefined, set `error` to null, set `pending` to false, set `status` to `idle`, and mark any currently pending requests as cancelled
```

Reply:
Perfect. Shall we add backticks to undefined, null, and false? Also, there's probably less value in telling the user about _marking_ as cancelled, and more in putting an emphasis on the actual result: '... and cancel any currently pending requests'.

Reply:
happy either way. the reason I wasn't wanting to do that as it doesn't actually stop them running, and a few queries have come in previously by people who want somehow to 'revoke' a promise and stop its execution

Reply:
Totally understandable, let's keep your version for clarity 👍